DATA_LAYER_INTEGRATION.md

```diff
@@ -0,0 +1,280 @@
+      "@type": "core-components-examples/components/image",
+      "repo:modifyDate": "2019-01-09T16:58:39Z",
+      "parentId": "page-df1699a779"
+    },
+    "accordion-3b90a50076": {
+      "shownItems": [
+        "accordion-3b90a50076-item-6b9d62c47d",
+        "accordion-3b90a50076-item-f05b6c6615",
+        "accordion-3b90a50076-item-f3fcd996f5",
+        "accordion-3b90a50076-item-cd527eac50"
+      ],
+      "@type": "core-components-examples/components/accordion",
+      "repo:modifyDate": "2020-01-14T16:59:24Z",
+      "parentId": "page-df1699a779"
+    },
+    "title-13869e3afb": {
+      "dc:title": "Standard",
+      "@type": "core/wcm/components/title/v2/title",
+      "repo:modifyDate": "2018-12-07T12:49:20Z",
+      "parentId": "page-df1699a779"
+    },
+    "text-c36e0b2cf9": {
+      "@type": "core/wcm/components/text/v2/text",
+      "repo:modifyDate": "2018-12-07T12:49:23Z",
+      "xdm:text": "<p>Default title without any configuration. The title is taken from the current page.</p>\n",
+      "parentId": "page-df1699a779"
+    }
+  }
+}
+```
+
+## Components supporting the Data Layer
+
+The following table shows the components supporting the data layer:
+
+Components | Data Layer Support
+---------- | -------------------
+Accordion | x
+Breadcrumb | x
+Button | x
+Carousel | x
+Container | 
+Content Fragment | 
+Content Fragment List | 
+Download | 
+Embed | 
+Experience Fragment | 
+Form button | 
+Form container | 
+Form hidden field | 
+Form options field | 
+Form text field | 
+Image | x
+Language Navigation | x
+List | x
+Navigation | x
+Page | x
+Progress Bar | 
+Quick Search | 
+Separator | 
+Sharing | 
+Tabs | x
+Teaser | x
+Text | x
+Title | x
+
+## Core Component Schemas
+
+#### Component and Container Item
+
+Schema used for all the components that are not listed below.
+
+```
+id: {                   // component ID
+    @type               // resource type
+    repo:modifyDate     // last modified date
+    dc:title            // title
+    dc:description      // description
+    xdm:text            // text
+    xdm:linkURL         // link URL
+    parentId            // parent component ID
+}
+```
+
+#### Container Components
+
+Schema used for the Accordion, the Carousel and the Tabs components.
+
+```
+id: {
+    @type
+    repo:modifyDate
+    dc:title
+    dc:description
+    xdm:text
+    xdm:linkURL
+    parentId
+    shownItems          // array of the displayed item IDs
+}
+```
+
+#### Page
+
+Schema used for the Page component:
+
+```
+id: {
+    @type
+    repo:modifyDate
+    dc:title
+    dc:description
+    xdm:text
+    xdm:linkURL
+    parentId
+    xdm:tags            // page tags
+    repo:path           // page path
+    xdm:template        // page template
+    xdm:language        // page language
+}
+```
+
+#### Image
+
+Schema used for the Image component:
+
+```
+id: {
+    @type
+    repo:modifyDate
+    dc:title
+    dc:description
+    xdm:text
+    xdm:linkURL
+    parentId
+    image
```

Comment:
I would add a comment here as well: ` // asset detail` to make it clear this is extra to the general component properties.

Reply:
Addressed in https://github.com/adobe/aem-core-wcm-components/pull/1026/commits/3aef3ebdf90b8c4c43161e1cb2ebfca204c5edd1